app/decorators/decidim/participatory_process_decorator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+#
+# This decorator adds required associations between Decidim::ParticipatoryProcess and Departaments.
+#
+#
+require_dependency 'decidim/participatory_process'
+Decidim::ParticipatoryProcess.class_eval do
+  belongs_to  :areas,
+              class_name: 'Areas',
+              foreign_key: 'decidim_department_id'
+
+  belongs_to  :participatory_process_group,
```

Comment:
This field already exists in the original participatory_process class:
- https://github.com/decidim/decidim/blob/c964194a0c8b79374f00bfdef1678c3457c8a523/decidim-participatory_processes/app/models/decidim/participatory_process.rb#L30
so, why do we need to add it again @spuigmitjadxc ?